src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -157,7 +157,14 @@ public class Thread implements Runnable {
 
     /* Fields reserved for exclusive use by the JVM */
     private boolean stillborn = false;
-    private long eetop;
+
+    /*
+     * Reserved for exclusive use by the JVM. The historically named
+     * `eetop` holds the address of the underlying VM JavaThread, and is set to
+     * non-zero when the thread is started, and reset to zero when the thread terminates.
+     * A non-zero value indicates this thread isAlive().
+     */
```

Comment:
Shortened comment. Ok.

Reply:
Yes. Mostly because FieldHolder is not in JDK 17, so mentioning it would be confusing, IMO.